src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall().toString());
+        Lottery ball2 = new Lottery();
```

Comment:
`Lottery` class can create many random balls, you don't need to create a new instance of `Lottery`

**ToDo:** 
1. please use one instance of `Lottery` class, to create 3 balls
2. Use loop to generate and print balls